Project.toml

```diff
@@ -10,10 +10,11 @@ InverseFunctions = "3587e190-3f89-42d0-90ee-14403ec27112"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Distributions = "0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25"
 Functors = "0.1, 0.2"
 InverseFunctions = "0.1.2"
-StatsFuns = "0.9.13"
```

Comment:
Older versions of StatsFuns tend to cause problems with LogExpFunctions and IrrationalConstants - StatsFuns did not depend on LogExpFunctions and IrrationalConstants and therefore it is possible to load older versions of StatsFuns that already define the irrationals and additionally LogExpFunctions and IrrationalConstants which also define them. These leads to many redefinitions.

Reply:
Alright I will put it back, I was trying this because of unrelated issues of incompatibilities because of `KernelFunctions` but that should not be a problem anymore